nntrainer/tensor/tensor.cpp

```diff
@@ -406,6 +378,21 @@ Tensor &Tensor::subtract(Tensor const &m, Tensor &out) const {
   return add(m, out, -1);
 }
 
+int Tensor::pow_i(float exponent) {
```

Comment:
Please rearrange in headers as well.

Reply:
This is synced with header. you mean moving to else where?

Reply:
I looked at previous commit.